hack/imports.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package hack
 
 /*
-Package imports imports dependencies required for "dep ensure" to fetch all of the go package dependencies needed
+Package imports dependencies required for "dep ensure" to fetch all of the go package dependencies needed
```

Comment:
`\* Package hack imports ...`

And, if you're already fixing this, can you move the package comment above the package?